studybuddies/src/app/app.module.ts

```diff
@@ -20,6 +22,10 @@ const routes: Routes = [
   { path: 'tutor/createClass', component: CreateClassTutorComponent },
   { path: 'tutor/classList', component: ClassListTutorComponent },
   { path: '', component: HomeComponent },
+  { path: 'errorPage', component: ErrorPageComponent },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
  { path: "errorPage", component: ErrorPageComponent },
```